tests/functional/pages/surveys/summary_screen/summary.page.js

```diff
@@ -27,5 +27,7 @@ class SummaryPage extends QuestionPage {
 
   testDecimalEdit() { return '[data-qa="test-decimal-edit"]'; }
 
+  summaryGroupTitle() { return '#summary-group'; }
+
```

Comment:
@nealedj Shouldn't this be regenerated instead of editing?

Reply:
Yep that was generated. See https://github.com/ONSdigital/eq-survey-runner/pull/1482/files#diff-4f96ce7885038bc4f8accbd23d9cb930R90